src/locales/en/pages.yaml

```diff
@@ -729,60 +729,64 @@ addendum-50:
-      Besides the regular rehearsals we have a rehearsal weekend twice a year. During these weekends we work hard on the current projects as well as party and have fun.
-      Krashna is not only a music association, but switches  into a sailing association once every two years. Other sport activities are practiced by our members such as soccer and beach volleyball. Krashna also participates in the annual inter orchestral soccer day.
-      When we near the summer holidays there is often a larger project. This year we went to Paris and Tours (France). Next year we will have the 11th lustrum project which promises to be an amazing experience.
-      All these different activities could not take place without the hard work of multiple committees. Being part of a committee is a fun way to get to know people and work on personal development which is strongly encouraged within the association. This way you get to know a lot of other members quickly.
-
-    ## What are we looking for?
-      We are looking for people who enjoy making music in small or larger groups. This can be done by singing in the choir, participating in chamber music events or by playing in the orchestra.
-      In the choir everybody is allowed to participate. No audition is required, however having fun is a must!
-
-      For the orchestra there are auditions. At the moment we are looking for the following instruments:
-      - Violin
-      - Viola
-      - Cello
-      - Double basses
-      - Horn
-      - Flute (1 permanent spot and one temporary spot until February)
+      The choir and orchestra both rehearse weekly. The orchestra rehearses on Mondays (7:30 PM - 10:30 PM) and the choir on Wednesdays (7:30 PM - 10:00 PM). You can join both if you like. Rehearsals take place at X, followed by a social drink.
+
+    ## Are there other activities besides rehearsals?
+      Yes, we have rehearsal weekends twice a year where we work hard on projects and have time for social activities and parties. Krashna also turns into a water sports association every two years during the sailing week. Members often organize other sports activities like volleyball and soccer. Krashna also participates in an annual inter-orchestral soccer day.
+      In the summer, we often have a big project or go on tour. This year, we visited Prague and Krakow.
+      These activities wouldn't be possible without our committees. Joining a committee is a great way to meet people and develop new skills, and it's highly encouraged in our association. This helps new members get involved quickly.
+
+    ## Who are we looking for?
+      We are looking for people who enjoy making music together. You can join the choir, participate in chamber music events, or play in the orchestra.
+      Everyone is welcome in the choir—no audition required. Just a love for singing.
+
+      For the orchestra, auditions are required. We are currently looking for the following instruments: TODO
```

Comment:
Leftover TODO. Instrument lists between English and Dutch don't match (that is probably what the TODO was for)

Reply:
Fixed the list :)